exchange/bitswap/workers.go

```diff
@@ -182,10 +182,11 @@ func (bs *Bitswap) rebroadcastWorker(parent context.Context) {
 	defer cancel()
 
 	broadcastSignal := time.After(rebroadcastDelay.Get())
+	tick := time.Tick(10 * time.Second)
 
 	for {
 		select {
-		case <-time.Tick(10 * time.Second):
```

Comment:
> I searched the codebase for any other similar mistakes and found none.

`time.Tick` creates a channel intended to be used repeatedly, it does some magic runtime stuff and inserts a system timer callback to send on the channel, so it doesnt show up as a goroutine.